HACKING.md

```diff
@@ -50,3 +52,106 @@ Overview of Mender client architecture.
 +------------------+       +------------------------+        +--------------------+
 
 ```
+
+## Hacking on Mender client locally
+
+Build mender with support for local development:
```

Comment:
I'm not sure about this. It seems sufficiently readable for me. Your proposal just makes it more verbose, something you want to avoid when providing instructions.